app/services/invitations/generate_letter.rb

```diff
@@ -54,6 +54,7 @@ def locals
         department: department,
         applicant: applicant,
         organisation: organisation,
+        organisation_independent_from_cd: organisation_independent_from_cd,
```

Comment:
niptick: on pourrait ne garder que `independent_from_cd` comme nom de variable, vu que ça ne fait pas forcément référence à l'`organisation` de la ligne du dessus.

Reply:
j'ai opté pour `independent_from_cd_message`

Reply:
J'ai des doutes sur le fait que `independent_from_cd_message` soit clair à lire ici hors du contexte de cette PR.

Reply:
better like this ?

Reply:
LGTM 👍